File: hash.js
import { readFile } from 'node:fs/promises'
import { createHash } from 'node:crypto'

const hashes = (await readFile('hashes.txt', { encoding: 'utf-8' })).split(
    /\r?\n/
)

const dictionaries = (
    await readFile('dictionary.txt', { encoding: 'utf-8' })
).split(/\r?\n/)

for (const hash of hashes) {
    for (const pw of dictionaries) {
        if (createHash('sha256').update(pw).digest('hex') === hash)
            console.log(pw)
    }
}

File: hash.py
import hashlib

with open('hashes.txt', 'r') as file:
    hashes = file.read().splitlines()

with open('dictionary.txt', 'r') as file:
    dictionary = file.read().splitlines()

for hash in hashes:
    for pw in dictionary:
        if hashlib.sha256(pw.encode()).hexdigest() == hash:
            print(pw)